recipes/pymodsecurity/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Instead of a `build.sh`, can you just use the `script` entry?

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

Also I thnk it would be preferable to use pip:

https://conda-forge.org/docs/meta.html#use-pip